examples/plot_multi_axes.py

```diff
@@ -0,0 +1,121 @@
+"""
+Test Plots with Multiple Axes in Matplotcheck
+=============================================
+
+Below you will find some examples of how to use MatPlotCheck
+to test Matplotlib objects with more than one axes in Python.
+
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# To begin, import the libraries that you need.
+
+import matplotcheck.base as pt
+import matplotlib.pyplot as plt
+import pandas as pd
+
+###############################################################################
+# Setup
+# -----
+# First you must create some example data to plot over multiple axes. This
```

Comment:
Change to: "First, you need some data to plot over multiple axes of a figure.  This data could be satellite imagery bands, calculated values of vegetation indices over time for multiple sites, or any other data for which you want to create multiple plots within one figure. In this example, two Pandas dataframes are created to represent different datasets that can be plotted as individual line plots."